ross-nichols-solutions.js

```diff
@@ -0,0 +1,154 @@
+//PART ONE
+//#1
+const calculateCube = (num) => {
+    return num * num * num
+  }
+  
+  calculateCube(5)
+  console.log(calculateCube(5))
+  
+  //#2
+  const isAVowel = (str) => {
+    if(str.includes('a')) {
+      return true
+    } else if (str.includes('e')){
+      return true
+    } else if (str.includes('i')) {
+      return true
+    } else if (str.includes('o')) {
+      return true
+    } else if (str.includes('u')) {
+      return true
+    } else if (str.includes('y')) {
+      return 'sometimes'
+    } else {
+      return false
+    }
+  }
```

Comment:
nice, and as a *language person* I appreciate the consideration you gave the letter 'y' 👍 ! how would you account for a capital letter, though?